src/Plugin.php

```diff
@@ -36,8 +36,9 @@ class Plugin {
   public static function override_load_textdomain($override, $domain, $mofile) {
     global $l10n, $l10n_unloaded;
     $l10n_unloaded = (array) $l10n_unloaded;
+    $cache_key = $domain . ':' . $mofile;
```

Comment:
After rebasing against latest master, we should change the `:filemtime` suffix to be appended to the cache key instead of the cache context/group?

Reply:
To me your approach to have 2 different groups, one for mo objects and one for timestamps for invalidation makes sense. I just added the `:` for clarity when reading the keys in redis (/whatever can be set up in many contexts)

Reply:
Conceptually, a group would be used to select and invalidate all items at once.

Unfortunately that is not supported by WP-CLI (yet) as `wp cache delete` always requires a key, but if it wouldn't, you'd be able to run `wp cache delete-group gettext` to delete all cache items at once.

As the filemtime belongs to the same group, it would be more correct to append it to the key name.

While being there, we could rename the group to `gettext`.